### ISSUE TITLE
Cancelling import process

### ISSUE THREAD

**Issue by ruzannakamalyan** [assimp/assimp#912]
Hi,

I couldn't find a way of cancelling the import process. Am I missing something ?

Any help is appreciated!

**Comment by turol** [comment]
You could set a progress handler and throw an exception from it. I haven't tested that though and am not sure if it's officially supported.

**Comment by kimkulling** [comment]
No, this is not an officially heature. But we could think about  defining some kind of a special exception to cancel the process.  At the moment we do not work multithreaded for this. We will have to adapt our API.

**Comment by ruzannakamalyan** [comment]
Thanks for the reply.

At the moment I'm trying to adapt the ProgressHandler's part to AssimpNet to follow @turol 's suggestion. Though having some difficulties on marshaling the class (newbie in this).

**Comment by kimkulling** [comment]
Sounds good, let's do it. Who wants to give it a try?

**Comment by JekaS178RUS** [comment]
Wait a minute, we can return false from Assimp::ProgressHandler::Update function to abort loading at the next possible occasion, as written in doxygen comment in hpp file.
Is it obsolete or I mess something?

**Comment by kimkulling** [comment]
Sound good to me. I guess that is the solution.
